src/hatti/views/table.cljs

```diff
@@ -82,9 +84,9 @@
 (defn formatter
   "Formatter for slickgrid columns takes row,cell,value,columnDef,dataContext.
    Get one with (partial formatter field language)."
-  [field language row cell value columnDef dataContext]
+  [field language name-or-label row cell value columnDef dataContext]
```

Comment:
again w/the name, we'd also need to change the call below if we change the signature as I've suggested above